modules/programs/ncspot.nix

```diff
@@ -5,17 +5,19 @@
   ...
 }:
 let
+  inherit (lib) mkIf;
```

Comment:
while I dont feel strongly about this, I wonder if @khaneliman is ok with that ? since he removed the `with lib`.

Reply:
Inherit is the static analysis supported method of doing it.